server/src/main/java/org/elasticsearch/action/datastreams/GetDataStreamAction.java

```diff
@@ -179,6 +181,7 @@ public static class DataStreamInfo implements SimpleDiffable<DataStreamInfo>, To
             public static final ParseField TEMPORAL_RANGES = new ParseField("temporal_ranges");
             public static final ParseField TEMPORAL_RANGE_START = new ParseField("start");
             public static final ParseField TEMPORAL_RANGE_END = new ParseField("end");
+            public static final ParseField TIME_SINCE_LAST_AUTO_SHARDING = new ParseField("time_since_last_auto_sharding");
```

Comment:
Super minor nit, but how do you feel about using `time_since_last_auto_shard_event` or `time_since_last_auto_sharding_event` to avoid making "sharding" into a verb?